lms/templates/pdf_viewer.html

```diff
@@ -27,7 +27,7 @@
 <html dir="ltr" mozdisallowselectionprint moznomarginboxes>
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
@cahrens I tested this on iOS and Android and was able to zoom both the document and the viewer controls on both devices.